src/_pytest/python.py

```diff
@@ -1189,6 +1209,8 @@ def parametrize(
             ``bool``, or ``None``.
             They are mapped to the corresponding index in ``argvalues``.
             ``None`` means to use the auto-generated id.
+            ``pytest.HIDDEN_PARAM`` means to hide the parameter set
+            from the test name.
```

Comment:
This section is shown here: https://pytest--13229.org.readthedocs.build/en/13229/reference/reference.html#pytest.Metafunc.parametrize

Although a bit hidden, perhaps it helps to show the version where this was added, and mentioning the restriction of it being used at most 1 time:

```suggestion

            .. versionadded: 8.4

            ``pytest.HIDDEN_PARAM`` means to hide the parameter set
            from the test name. Can only be used at most 1 time, as
            test names need to be unique.
```

Also `HIDDEN_PARAM` should appear under the `Constants` at the top of the page, and then the `pytest.HIDDEN_PARAM` should be a link to it.

Also it should be mentioned here: https://pytest--13229.org.readthedocs.build/en/13229/reference/reference.html#pytest-param

Reply:
I could not get Sphinx to display `versionadded` only for a subsection of `id` and `ids` parameters. It's probably enough to display it on `pytest.HIDDEN_PARAM` itself.